src/graph_notebook/magics/graph_magic.py

```diff
@@ -251,6 +263,46 @@ def graph_notebook_config(self, line='', cell='', local_ns: dict = None):
 
         return self.graph_notebook_config
 
+    @line_cell_magic
+    def neptune_config_allowlist(self, line='', cell=''):
+        parser = argparse.ArgumentParser()
+        parser.add_argument('mode', nargs='?', default='add',
+                            help='mode (default=add) [add|remove|overwrite|reset]')
+        args = parser.parse_args(line.split())
+
+        try:
+            cell_new = ast.literal_eval(cell)
+            input_type = 'list'
+        except:
+            cell_new = cell
+            input_type = 'string'
+
+        allowlist_modified = True
+        if args.mode == 'reset':
+            self.neptune_cfg_allowlist = copy(NEPTUNE_CONFIG_HOST_IDENTIFIERS)
+        elif cell != '':
+            if args.mode == 'add':
```

Comment:
Dont we need a list operation as well?

Reply:
Running the `%neptune_config_allowlist` line magic with no arguments will display the list. This is consistent with the behavior of other hybrid line/cell magics like `%graph_notebook_config`.